### PR TITLE
Update apps.json

### PR DESCRIPTION
**Pull request by thivalle**
List of 10 sites that use Big Bang Shop
https://www.leschemises.com.br/
https://www.lojavirtualbacker.com.br/
https://www.klus.com.br/
https://www.bamaqconsorcio.com.br/
https://gifties.store/
https://www.goodiesinabox.com.br/
https://www.dashuniformes.com.br/
https://www.carroavenda.com.br/
https://zak.com.br/
https://www.lettebeachwear.com.br/

Thanks

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -1112,6 +1112,16 @@
       "implies": "Python",
       "website": "http://docs.python.org/2/library/basehttpserver.html"
     },
+    "BigBangShop": {
+      "cats": [
+        6
+      ],
+      "headers": {
+        "X-SERVER": "BIGBANGSHOP*\w+"
```
Comment (jvoisin) [inline src/apps.json]: What about `"X-SERVER": "^BIGBANGSHOP" ` instead?

Comment (thivalle) [reply src/apps.json]: My server responds a header with the following information:
X-SERVER: BIGBANGSHOP HOST 1
however, this response may vary according to the node accessed.
ex: X-SERVER: BIGBANGSHOP HOST 2
X-SERVER: BIGBANGSHOP HOST 3


I'm not very good at regular expression

Comment (thivalle) [reply src/apps.json]: my servers also respond to a cookie with the name "BIG_BANG_SHOP_SID"
how can i use it?

Thanks

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**